src/site/_data/paths/animations.json

```diff
@@ -0,0 +1,43 @@
+{
+  "slug": "animations",
+  "cover": "/images/collections/animations.svg",
+  "title": "Animations",
+  "updated": "September 25, 2020",
+  "description": "Techniques for developing high-performance CSS animations.",
+  "overview": "To do.",
+  "topics": [
+    {
+      "title": "Concepts",
+      "pathItems": [
+        "animations-overview"
+      ]
+    },
+    {
+      "title": "Guides",
+      "pathItems": [
+        {
+          "title": "Animation basics",
+          "url": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations"
+        },
```

Comment:
(Re-adding a comment that was closed accidentally)

@robdodson this change requires pathItems to either accept a string or an object with the properties title and URL

Reply:
I left this as-is. LMK if you wanted me to make a change, @robdodson.

> Worth noting that we likely won't be able to machine translate those title strings when we ship other languages.

Cross that bridge when we get there?

> It's more work but a possible alternative would be to make it so every pathItem is an object, that object would either have a slug or url/title properties. Then we'd check for those throughout the codebase.

Sounds like you're saying to convert all `src/_data/paths` JSON files to objects like this? Do you mean we should do that in this PR or is it approach to consider in the future?